https://issues.redhat.com/browse/ACM-20979--adding new topic WIP (#8128)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/backup_hcp.adoc

[#config-hcp-backup]
= Backup and restore for hosted control planes and hosted clusters

> The {ocp-short} {hcp} disaster recovery documentation describes how to install the {oadp} operator, how to create {oadp} resources, and how to use the {oadp} to backup and restore hosted clusters. See the following additional information about backup and restore for hosted clusters that applies to {acm-short}:

Reviewer: This is a little anthropomorphic, and takes the action away from the user. What do you think of something similar to this? 
```suggestion
See the {ocp-short} {hcp} disaster recovery documentation to help you install the {oadp} operator, create {oadp} resources, and use the {oadp} to backup and restore hosted clusters. For more help, see the documentation backup and restore for hosted clusters that applies to {acm-short}:
```

> * Restore the hosted clusters before you restore the {acm-short} hub cluster. Find the process to restore hosted clusters at link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/hosted_control_planes/high-availability-for-hosted-control-planes#hcp-disaster-recovery-oadp[OpenShift Hosted Control Plane Disaster Recovery documentation]. However, you do not need to install the {oadp} operator in the `openshift-adp` namespace.

Reviewer: ```suggestion
* Restore the hosted clusters before you restore the {acm-short} hub cluster. For instructions, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/hosted_control_planes/high-availability-for-hosted-control-planes#hcp-disaster-recovery-oadp[OpenShift Hosted Control Plane Disaster Recovery documentation]. 
```

Reviewer: If you think we nee the content "however, you do not need to install the oadp operator" then keep it. I am not used to seeing a "however" type of note in our bullet lists. 

Maybe it would read better as a literal note? 

**Note:** You do not nee to install the odap operator. 

It seems like this type of note would be better in the actual documenation for the hosted control plane, but I understand we can't control that. I'm not sure if it's even needed here.

> * For {acm-short}, you can enable backup and restore and use the {oadp} operator that is installed by the backup and restore component. 

Reviewer: The previous bullet doesn't use the sentence structure, "you can restore..." so for consistency, I think we can omit the "you can" 

```suggestion
* For {acm-short}, enable backup and restore and use the {oadp} operator that is installed by the backup and restore component. 
```

* {oadp} resources, such as `DataProtectionApplication`, `BackupSchedule`, `Backup` and `Restore` for your hosted clusters, are created in the `open-cluster-management-backup` namespace.

> * See the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/hosted_control_planes/high-availability-for-hosted-control-planes#hcp-backup-restore-aws-overview_hcp-disaster-recovery-aws[Overview of Hosted Cluster backup and restore process] for more information about the backup and restore process.

Reviewer: ```suggestion
* For more information, see the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/hosted_control_planes/high-availability-for-hosted-control-planes#hcp-backup-restore-aws-overview_hcp-disaster-recovery-aws[Overview of Hosted Cluster backup and restore process].
```
